.circleci/config.yml

```
@@ -40,7 +40,11 @@ jobs:
           name: Export package version
           command: |
             PKG_VERSION=$(node -e "console.log(require('./package.json').st2_version);")
-            PKG_RELEASE=$(packagecloud.sh next-revision xenial ${PKG_VERSION} st2web)
+            if [ -n "$PACKAGECLOUD_TOKEN" ]; then
+              PKG_RELEASE=$(packagecloud.sh next-revision xenial ${PKG_VERSION} st2web)
+            else
+              PKG_RELEASE=1
```

Following the  https://github.com/StackStorm/st2-packages/blob/master/.circle/buildenv_st2.sh#L49-L50 it's best to preserve any useful code comments. 
```suggestion
              # is fork
              PKG_RELEASE=1
```
This is actually how it was helpful to find the fix based on st2-packages repo code.